tests/integration/auth_utils.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+from base64 import b64encode
+from os.path import join
+from secrets import token_urlsafe
+from urllib.parse import urlencode
+
+import requests
+
+
+def construct_authorization_url(hydra_url: str, client_id: str, client_secret: str) -> str:
```

Comment:
Nitpick:

- Better align with other `get_xxx` functions for naming? Like `get_authorization_url`.